test/rulesets/Base/arraymath.jl

```diff
@@ -148,4 +148,12 @@
         test_rrule(-, A)
         test_rrule(-, Diagonal(A); output_tangent=Diagonal(Ā))
     end
+
+    @testset "addition" begin
+        A = randn(4, 4)
+        B = randn(4, 4)
+        C = randn(4, 4)
+
+        test_rrule(+, A, B, C)
+    end
```

Comment:
And a test with trailing dimensions:
```suggestion
    @testset "addition" begin
        test_rrule(+, randn(4, 4), randn(4, 4), randn(4, 4))
        test_rrule(+, randn(3), randn(3,1), randn(3,1,1))
    end
```